### PR TITLE
8.2XS82ECU1015

### PR DESCRIPTION
**Pull request by gduperrey**

### CODE REVIEW

**Review by benjamreis — APPROVED** [review]

**Review by Wescoeur — APPROVED** [review]

**Review by andSmv — APPROVED** [review]

File: SPECS/xen.spec [hunk 1]
```diff
@@ -1131,6 +1153,11 @@ touch %{_rundir}/reboot-required.d/%{name}/%{version}-%{base_release}
 
 %{?_cov_results_package}
 %changelog
+* Wed Aug 10 2022 Gael Duperrey <gduperrey@vates.fr> - 4.13.4-9.25.1
+- Synced from hotfix XS82ECU1015
+- Remove amd-iommu-correct-xt-handling.patch, it's allready in the XS82ECU1015
```
Comment (stormi) [inline SPECS/xen.spec:1158]:
```suggestion
- Remove amd-iommu-correct-xt-handling.patch, it's already in XS82ECU1015
```

Comment (stormi) [reply SPECS/xen.spec:1158]: /!\ last time I attempted to amend a merge commit, this didn't go well.

File: SPECS/xen.spec [hunk 2]
```diff
@@ -19,9 +19,9 @@ Version: 4.13.4
 # the xen_extra field can't hold more than 16 chars
 # so instead of using %%release to define XEN_VENDORVERSION
 # we create a base_release macro, that doesn't contain the dist suffix
-%define base_release 9.24.2
+%define base_release 9.25.1
 Release: %{base_release}%{?dist}
-License: GPLv2 and LGPLv2+ and BSD
+License: GPLv2 and LGPLv2+ and BSD and MIT and Public Domain
```
Comment (stormi) [inline SPECS/xen.spec:24]: BSD is not mentioned in the upstream spec file anymore

File: SPECS/xen.spec [hunk 3]
```diff
@@ -19,9 +19,9 @@ Version: 4.13.4
 # the xen_extra field can't hold more than 16 chars
 # so instead of using %%release to define XEN_VENDORVERSION
 # we create a base_release macro, that doesn't contain the dist suffix
-%define base_release 9.24.2
+%define base_release 9.25.1
 Release: %{base_release}%{?dist}
-License: GPLv2 and LGPLv2+ and BSD
+License: GPLv2 and LGPLv2+ and MIT and Public Domain
```
Comment (stormi) [inline SPECS/xen.spec:24]: This still differs from upstream's License tag. Don't forget to compare with the XS-8.2 branch and make sure all the changes are relevant.

**Review by stormi — APPROVED** [review]

**Pull request merged**